### ISSUE TITLE
Lightning

### ISSUE THREAD

**Issue by rschwarzjr** [bruhautomation/ESP-MQTT-JSON-Digital-LEDs#34]
If you have more than 255 leds the lighting effect doesn't work properly. If random16 is changed for random8 it fixes the problem (unless you have 65,000 leds I guess)

**Comment by eldoner** [comment]
Do you mean all `random8` in _EFFECT LIGHTING_ section? Because I have that issue too with this effect.

**Comment by rschwarzjr** [comment]
Yes. I went through all the code and changed all random8 to random16 when it is used to pick a specific led and it fixed most of my problems. Not just lightning.
